daemon/container.go

```diff
@@ -649,12 +637,11 @@ func (container *Container) ReleaseNetwork() {
 	if container.Config.NetworkDisabled || !container.hostConfig.NetworkMode.IsPrivate() {
 		return
 	}
-	eng := container.daemon.eng
 
-	job := eng.Job("release_interface", container.ID)
-	job.SetenvBool("overrideShutdown", true)
-	job.Run()
-	container.NetworkSettings = &NetworkSettings{}
+	_ = bridge.Release(container.ID)
+	//job.SetenvBool("overrideShutdown", true)
```

Comment:
I don't know anything about this part.  I cannot think of the usecase for it right now.